README.md

```diff
@@ -15,6 +15,8 @@ Contact us at <support@chartiq.com> to request sample code and guidance on how t
 
 - iOS 12.0 or later.
 
+**Important:** This SDK requires additional configuration to be able to work with the remote npm registry introduced in version 9.5.1.
```

Comment:
Do we want to say version 9.5.1 and not 9.6.0? Technically 9.5.1 exists, but it was never documented on the release notes.

Reply:
In the end it probably doesn't matter. If a client gets confused we can just give them 9.6.

Reply:
I discovered release notes and upgrade instructions for 9.5.1 that were created by Rob in Master, but they never made it to the docs site. They will when 9.6 goes live unless Gus/ you think otherwise.